gui_views/state.py

```diff
@@ -12,14 +12,17 @@ def __init__(self, instances, catalog):
         self.drag_state = True
         self.draw_wire_state = False
         self.sym_objects = {} # Map name to actual symobject (has coords)
-        self.current_sym_object = None
+        self.selected_sym_objects = []
         self.line_drawer = None
         self.scene = None
         self.mainWindow = None
         self.instances = instances
         self.catalog = catalog
         self.buttonView = None
         self.fileName = None
+        self.copyState = False
+        self.selectedObject = None
```

Comment:
Do we still need self.selectedObject? I think I added it for single object copy/pasting but I see we're using lists now.